test/runner.py

```diff
@@ -2,6 +2,9 @@
 import os
 import sys
 
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
this was kicked out in a previous PR, why add it again?

if it's really needed (it shouldn't), please add a clear comment here